locales/en-US/messages.properties

```diff
@@ -280,10 +280,11 @@ contributeReason1=Your work impacts users all over the world
 contributeReason2=Gain valuable experience and mentorship
 contributeReason3=Support Mozilla's mission of an open internet for all
 contributeReason4=It feels good!
-
-getInvolvedDescription1=Thimble is an <a href="https://en.wikipedia.org/wiki/Open-source_software">Open Source</a> project created by the <a href="https://www.mozilla.org/foundation/">Mozilla Foundation</a> in partnership with <a href="http://cdot.senecacollege.ca/">CDOT at Seneca College</a>. Since the beginning, contributors from all over the world have played an important role in making it a success.
+wikipediaOpenSourceURL=https://en.wikipedia.org/wiki/Open-source_software
+mozillaFoundationURL=https://www.mozilla.org/foundation/
```

Comment:
Okay, I think I misunderstood your original request.

* So, we don't want any of the URLs as separate translatable strings in the properties file
* We want the wikipedia link & URL hard-coded into the string in the properties file
* We want the other two URLs (seneca / foundation) to be defined as variables in the template and passed in (but not translatable)

Is that correct?